python-threatexchange/threatexchange/hashing/pdq_faiss_matcher.py

```diff
@@ -53,6 +53,7 @@ def search(
         queries: t.Sequence[PDQ_HASH_TYPE],
         threshhold: int,
         return_as_ids: bool = False,
+        **kwargs
```

Comment:
(blocking?): I generally recommend against this pattern - this means that typos on calling the search method will be ignored, unless you always pass **kwargs into something else. You sometimes see this in older code when they want to force you use a keyword argument, but there is new syntax for that (`foo(a, *, kw_only)`)